request-builder.js

```diff
@@ -5,10 +5,17 @@ const _ = require('underscore');
 function buildSlots(slots) {
   const res = {};
   _.each(slots, (value, key) => {
-    res[key] = {
-      name: key,
-      value: value
-    };
+    if( _.isString(value)){
+      res[key] = {
+        name: key,
+        value: value
+      }
+    }else{
+      res[key] = {
+        name: key,
+        ...value
```

Comment:
Should we control if value contains the property `name` ? in the off-chance that it collides with our property `name`? another solution would be to put the values inside a `values` property, instead of the top-most level. something like:

```js
res[key] = {
  name: key,
  values: ...value
}
```

Reply:
What I want to do for now is to set the value of the property `resolutions`.
```
{
  name: 'key',
  value:'value',
  resolutions:'value1',
}
```

So I think we have 3 options below.
1. Users always have to set the property `name`.
```
res[key] = value
```
2. Users don't have to set the property `name`. but if they set the property `name`, it overrides the value of the property `name`.
```
res[key] = {
       name: key,
        ...value
}
```
3. Users don't have to set the property `name`.If they set the property `name`, it is ignored.
```
res[key] = {
        ...value,
       name: key
}
```
How do you think about these options?

Reply:
I like the 3rd option the best as it's the "safest". Thanks for the explanation! Sorry in my previous comment I missed the fact that we don't get to chose at what level we put the `values`, as we have to follow Alexa's structure.